tests/benchmarks/multicore/idle_with_pwm/testcase.yaml

```diff
@@ -0,0 +1,44 @@
+common:
+  sysbuild: true
+  tags: ci_build ci_tests_benchmarks_multicore
+  platform_allow:
+    - nrf54h20dk/nrf54h20/cpuapp
+  integration_platforms:
+    - nrf54h20dk/nrf54h20/cpuapp
+
+tests:
+  benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.no_sleep:
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "Multicore idle_with_pwm test on"
+        - "Multicore idle_with_pwm test iteration 0"
+        - "Multicore idle_with_pwm test iteration 1"
+
+  benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.idle:
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+      idle_with_pwm_CONF_FILE=prj_s2ram.conf
+      remote_CONF_FILE=prj_s2ram.conf
+      idle_with_pwm_CONFIG_TEST_SLEEP_DURATION_MS=500
+      remote_CONFIG_TEST_SLEEP_DURATION_MS=500
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_placeholder"
+
+  benchmarks.multicore.idle_with_pwm.nrf54h20dk_cpuapp_cpurad.s2ram:
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+      idle_with_pwm_CONF_FILE=prj_s2ram.conf
+      remote_CONF_FILE=prj_s2ram.conf
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_placeholder"
```

Comment:
Please add tags: ppk_power_measure for configuration which should use PPK

Reply:
Similar to https://github.com/nrfconnect/sdk-nrf/pull/17394

Reply:
Done, see line just below configuration name (~37).